server/superdesk/data_consistency/compare_repositories_tests.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import superdesk
+from eve.utils import ParsedRequest
+from superdesk.tests import TestCase
+from superdesk import get_resource_service
+from superdesk.data_consistency.compare_repositories import CompareRepositories
```

Comment:
It's a good strategy to import the class under test in a test class itself rather than at the test module level. The rationale behind it is that a failed import should not prevent the tests from being run, it should result in these tests' failures instead.

Just a suggestion.